### PR TITLE
Add Northstar landing page

### PR DESCRIPTION
**Pull request by solazio**
## Done

Add Northstar landing page

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [/engage/North-Americas-first-autonomous-runway-snowplough-case-study](http://0.0.0.0:8001/engage/North-Americas-first-autonomous-runway-snowplough-case-study)
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Make sure it complies with the [brief](https://docs.google.com/document/d/171v3o0-5sS_3Lii-KzwiYv83Uz2iRQATl6cknYF7qKE/edit?ts=5ced49ee#)

## Issue / Card

Fixes https://github.com/canonical-web-and-design/web-squad/issues/1164

### CODE REVIEW

**Comment by anasereijo** [comment]
- [ ]  the heading needs a background. please use the generic `light-grey` one. [check the guide](https://www.ubuntu.com/engage/_heading-help)

- [ ] the title is too long, I will ask Wendy to check it once she's back from a meeting

- [ ] the "Read about the complex software..." is too long to be `h4` please make it `p`

- [ ] can we have the green button on mobile?
Screenshot 2019-05-29 at 10 30 40

**Comment by anasereijo** [comment]
@WendyS1000 please let us know once the copy changes are done

**Comment by anasereijo** [comment]
@solazio logo here: https://assets.ubuntu.com/v1/bdbffc9a-northastar-logo.png

**Comment by anasereijo** [comment]
@SirSamTumless can you ask Marketing to provide us with an SVG version of Northstar logo so we can update this page sometime in the future? thanks

**Comment by anasereijo** [comment]
@solazio text was updated please check copydoc

**Comment by WendyS1000** [comment]
Can I review this? Please send me the url to review because I/m not sure how to get it from GitHub. Thanks!

**Comment by anasereijo** [comment]
@WendyS1000 here you go https://www-ubuntu-com-canonical-web-and-design-pr-5214.run.demo.haus/engage/North-Americas-first-autonomous-runway-snowplough-case-study

**Comment by anasereijo** [comment]
@solazio please make the logo 340px wide and centre it within the columns

**Comment by WendyS1000** [comment]
@anasereijo, @SirSamTumless @solazio this is good to go live, thank you very much.

File: templates/engage/North-Americas-first-autonomous-runway-snowplough-case-study.html [hunk 1]
```diff
@@ -0,0 +1,52 @@
+{% extends "engage/base_engage.html" %}
+
```
Comment (sowasred2012) [inline templates/engage/North-Americas-first-autonomous-runway-snowplough-case-study.html]: There should be a `{% block title %}` here, the page title is currently "| Ubuntu"

File: templates/engage/North-Americas-first-autonomous-runway-snowplough-case-study.html [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+{% block meta_copydoc %}https://docs.google.com/document/d/1uxXjTwzzhOQK7x9xpppWgRNVPLpnYA5BN1Mr1Zp2rHI/edit?ts=5ce644a0#heading=h.ud7r143one49{% endblock meta_copydoc %}
+
+{% block content %}
+
+<section class="p-strip--light">
+  <div class="row u-equal-height">
+    <div class="col-7 u-vertically-center">
+      <div>
+      <h1 class="u-no-margin--bottom u-sv3">
```
Comment (sowasred2012) [inline templates/engage/North-Americas-first-autonomous-runway-snowplough-case-study.html]: Tiny tweak - lines 13 to 22 should be indented one more level

**Comment by WendyS1000** [comment]
@anasereijo, @SirSamTumless @solazio  could we please push the Northstar case study live now with the shortened url and let me know when its live? We are including it in our May newsletter - it's going out tomorrow, but we're working on the text now. Thanks!

**Comment by solazio** [comment]
@WendyS1000 & @SirSamTumless this is now live.

**Pull request merged**